locales/en.json

```
@@ -2,67 +2,67 @@
+    "app_no_upgrade": "Everything is up-to-date",
+    "app_not_upgraded": "These apps were not upgraded: {apps}",
     "app_not_correctly_installed": "{app:s} seems to be incorrectly installed",
-    "app_not_installed": "The application '{app:s}' is not installed. Here is the list of all installed apps: {all_apps}",
+    "app_not_installed": "The application '{app:s}' is not to be found among all installed apps: {all_apps}",
     "app_not_properly_removed": "{app:s} has not been properly removed",
     "app_package_need_update": "The app {app} package needs to be updated to follow YunoHost changes",
-    "app_removed": "{app:s} has been removed",
+    "app_removed": "{app:s} removed",
     "app_requirements_checking": "Checking required packages for {app}…",
-    "app_requirements_failed": "Unable to meet requirements for {app}: {error}",
+    "app_requirements_failed": "Did not meet requirements for {app}: {error}",
     "app_requirements_unmeet": "Requirements are not met for {app}, the package {pkgname} ({version}) must be {spec}",
-    "app_sources_fetch_failed": "Unable to fetch sources files, is the url correct?",
+    "app_sources_fetch_failed": "Could not fetch sources files, is the URL correct?",
     "app_start_install": "Installing application {app}…",
     "app_start_remove": "Removing application {app}…",
-    "app_start_backup": "Collecting files to be backuped for {app}…",
+    "app_start_backup": "Collecting files to be backed up for {app}…",
     "app_start_restore": "Restoring application {app}…",
     "app_unknown": "Unknown app",
     "app_unsupported_remote_type": "Unsupported remote type used for the app",
-    "app_upgrade_several_apps": "The following apps will be upgraded : {apps}",
-    "app_upgrade_app_name": "Now upgrading app {app}…",
-    "app_upgrade_failed": "Unable to upgrade {app:s}",
-    "app_upgrade_some_app_failed": "Unable to upgrade some applications",
-    "app_upgraded": "{app:s} has been upgraded",
+    "app_upgrade_several_apps": "The following apps will be upgraded: {apps}",
+    "app_upgrade_app_name": "Now upgrading {app}…",
+    "app_upgrade_failed": "Could not upgrade {app:s}",
+    "app_upgrade_some_app_failed": "Some applications could not be upgraded",
+    "app_upgraded": "{app:s} upgraded",
     "apps_permission_not_found": "No permission found for the installed apps",
-    "apps_permission_restoration_failed": "Permission '{permission:s}' for app {app:s} restoration has failed",
-    "appslist_corrupted_json": "Could not load the application lists. It looks like {filename:s} is corrupted.",
-    "appslist_could_not_migrate": "Could not migrate app list {appslist:s}! Unable to parse the url… The old cron job has been kept in {bkp_file:s}.",
-    "appslist_fetched": "The application list {appslist:s} has been fetched",
+    "apps_permission_restoration_failed": "Grant the permission permission '{permission:s}' to restore {app:s}",
```

```suggestion
    "apps_permission_restoration_failed": "Could not restore the permission '{permission:s}' while restoring application {app:s}",
```

(though many messages related to permission will change in https://github.com/YunoHost/yunohost/pull/795 :/ )

---

How I read this is a permission is needed to restore the app, meaning one could grant said permission, and the app would be restored. The new active suggestion instructs the user in solving the problem. Your change seems to question the permission as the only thing that could not be restored, but does not detail why, or what to do about it.

---

Uuuuh not sure to understand your point but anyway this string is to be removed in https://github.com/YunoHost/yunohost/pull/795 :| ...